dev/NavigationView/NavigationView_InteractionTests/NavigationViewTests.cs

```diff
@@ -2972,6 +2961,51 @@ public void BackToolTipTest()
             }
         }
 
+        [TestMethod]
+        [TestProperty("TestSuite", "D")]
+        public void CloseToolTipTest()
+        {
+            if (PlatformConfiguration.IsDevice(DeviceType.Phone))
+            {
+                Log.Warning("Test is disabled on mobile due to lack of tooltips.");
+                return;
+            }
+
+            using (var setup = new TestSetupHelper(new[] { "NavigationView Tests", "NavigationView Test" }))
+            {
+                var panelDisplayModeComboBox = new ComboBox(FindElement.ByName("PaneDisplayModeCombobox"));
+
+                Log.Comment("Set PaneDisplayMode to LeftMinimal");
+                panelDisplayModeComboBox.SelectItemByName("LeftMinimal");
+                Wait.ForIdle();
+
+                var isPaneOpenCheckBox = new CheckBox(FindElement.ById("IsPaneOpenCheckBox"));
+
+                Log.Comment("Open the pane");
+                isPaneOpenCheckBox.Check();
+                Wait.ForIdle();
+
+                Verify.AreEqual(ToggleState.On, isPaneOpenCheckBox.ToggleState);
```

Comment:
Pass a comment string to the AreEqual method. Otherwise the logged error will just be "Failed: Verify.AreEqual(ToggleState.On, ToggleState.Off)", which is not easily understandable. Since Azure DevOps only shows the error string without context it is even harder to understand the failure reason.